multy_test/utility.cpp

```diff
@@ -108,6 +108,25 @@ bool blockchain_can_derive_address_from_private_key(Blockchain blockchain)
     return false;
 }
 
+std::string json_minified(std::string json_with_space)
```

Comment:
Function should take `const std::string&`, also it makes sense to rename argument to `input_json`

Reply:
Please add few tests that verify correctness of this. Test cases should include escape sequences:
`" \" "` == `" \" "`
`" \\ "` == `" \\  "`
and so on